index.html

```diff
@@ -186,6 +186,23 @@ <h1>I’m Roodjino Glad to see you!</h1>
         
 
         </section>
+        <form action="https://formspree.io/f/mpznorea" method="post">
+            <h2>Contact me</h2>
+            <p>If you have an application 
+                you are interested in developing, 
+                a feature that you need built or 
+                a project that needs coding. 
+                I’d love to help with it </p>
+            <ul>
+                <li><input type="text" id="name" name="name" placeholder="Enter your Name" required minlength="2" maxlength="30"></li>
+                <li><input type="email" id="email" placeholder="Enter your Email"  minlength="2" required ></li>
+                <li><textarea name="textarea" id="message" placeholder="Write your Message here" required  minlength="2" maxlength="500">Write your message here</textarea></li>
```

Comment:
- [ ] **Please remove the text which is hardcoded as Textarea value** (<textarea>Remove the text here</textarea>) That means the value inside the textarea tag. The placeholder attribute value already covers the role to direct the user about what to do 😃 👍